frontend/src/actions/locations.js

```diff
@@ -0,0 +1,17 @@
+import uuid from 'uuid';
+
+// ADD_LOCATIONS
+export const addLocations = (locations) => ({
+  type: 'ADD_LOCATIONS',
+  locations: locations.map(location => {
+      return {
+        key: uuid(),
+        project_id: location.project_ids[0],
```

Comment:
Check if `location.project_ids` & `location.project_types` is an Array before assigning.